### ISSUE TITLE
Contact

### ISSUE THREAD

**Issue by nicofr69** [LibGdxCiSample/test#13]
@yegor256
Just a message via GitHub to get in touch

**Comment by yegor256** [comment]
@nicofr69 yeah, I'm here :)

**Comment by nicofr69** [comment]
@yegor256 
Hi,
My understanding is that the Gradle build executed by Travis CI will create APK files in the Docker images, and that Docker images are deleted after the build.
Before deletion, I want to save the APKs, e.g. on GitHub, to manually sign and zipalign them on my PC before manually deploying them to Google Play (alpha channel).
Any idea how to do so?
I have tried adding deploy statements to .travis.yml:
deploy:
  provider: releases
  api_key: $GH_API_KEY
  file: |-
        - "android-debug.apk"
        - "android-release-unsigned.apk"
  skip_cleanup: true
  on:
    tags: true
     tags: true
But it doesn't work, something is misses.
Is there an alternative with "ATrultor deploy"?
Thanks

**Comment by yegor256** [comment]
@nicofr69 yes, it's possible. Let's create a new ticket, which will explain what doesn't work now and what has to be achieved. The description of the ticket will define its goal. And I'll estimate it and fix. Are you sure you want to do it all in the "test" project? Why not a real one?

**Comment by nicofr69** [comment]
@yegor256 The libgdx directory structure will be the same for the target project, so an open source (free) will be sufficient to setup the CI/CD process. Are you telling me to opt for a private project, or just for a more realistic project?

**Comment by yegor256** [comment]
@nicofr69 I would rather work with a private project, a real one. To save time and money. You can give me access there and we'll do exactly the same, but with real data/code.

**Comment by yegor256** [comment]
@nicofr69 also, keep in mind that CI/CD configuration for a private Github project is different than for a public one.

**Comment by nicofr69** [comment]
@yegor256 Ok, you convinced me, I will setup a private project.
